apps/docs/components/preview/TypographyPreview.tsx

```diff
@@ -0,0 +1,22 @@
+import "@hopper-ui/tokens/fonts.css";
```

Comment:
ce component là sert à rien, tu pourrais l'inline: 

au lieu de 
```
                    <td className="hd-table__cell hd-typo__cell" rowSpan={4}>
                        <TypographyPreview style={{ ...previewAdditionalStyles }} values={{ lineHeight, fontSize, fontWeight, fontFamily }} />
                    </td>
```
 tu aurais 
```
                    <td className="hd-table__cell hd-typo__cell" rowSpan={4}>
                        <div className="hd-preview hd-preview--font hd-preview--typography" style={{ lineHeight, fontSize, fontWeight, fontFamily, 
 ...previewAdditionalStyles }} >Aa</div>
                    </td>
```

Reply:
aussi, probablement qu'un span serait mieux que div

Reply:
preview.css is still useful, I could import it in my component but it feels weird.

Reply:
Je comprend je voyais pas le Preview.ts. Correct comme ça alors